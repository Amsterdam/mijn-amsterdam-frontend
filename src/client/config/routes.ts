import { generatePath, matchPath } from 'react-router-dom';

import { AppRoute, AppRoutes } from '../../universal/config/routes';
import { Match } from '../../universal/types';

export const AppRoutesRedirect = [
  {
    from: '/burgerzaken/document/:id',
    to: AppRoutes['BURGERZAKEN/IDENTITEITSBEWIJS'],
  },
  {
    from: '/stadspas',
    to: AppRoutes.HLI,
  },
  {
    from: '/stadspas/saldo/:pasnummer',
    to: AppRoutes.HLI,
  },
  {
    from: '/inkomen-en-stadspas/bijstandsuitkering/:id',
    to: AppRoutes['INKOMEN/BIJSTANDSUITKERING'],
  },
  {
    from: '/inkomen-en-stadspas/uitkeringsspecificaties/jaaropgaven',
    to: generatePath(AppRoutes['INKOMEN/SPECIFICATIES'], {
      kind: 'jaaropgave',
    }),
  },
  {
    from: '/inkomen/uitkeringsspecificaties/jaaropgaven',
    to: generatePath(AppRoutes['INKOMEN/SPECIFICATIES'], {
      kind: 'jaaropgave',
    }),
  },
  {
    from: '/inkomen-en-stadspas/uitkeringsspecificaties/',
    to: generatePath(AppRoutes['INKOMEN/SPECIFICATIES'], {
      kind: 'uitkering',
    }),
  },
  {
    from: '/inkomen/uitkeringsspecificaties/',
    to: generatePath(AppRoutes['INKOMEN/SPECIFICATIES'], {
      kind: 'uitkering',
    }),
  },
  {
    from: '/inkomen-en-stadspas/tozo/:version/:id',
    to: AppRoutes['INKOMEN/TOZO'],
  },
  { from: '/inkomen-en-stadspas', to: AppRoutes.INKOMEN },
];

export const PublicRoutes: string[] = [
  AppRoutes.API_LOGIN,
  AppRoutes.API1_LOGIN,
  AppRoutes.API2_LOGIN,
  AppRoutes.ACCESSIBILITY,
  AppRoutes.BFF_500_ERROR,
  AppRoutes.ACCESSIBILITY,
  AppRoutes.GENERAL_INFO,
  AppRoutes.HOME,
];

export const PrivateRoutes = Object.values(AppRoutes).filter(
  (path: string) => !PublicRoutes.includes(path)
);

export interface TrackingConfig {
  profileType: ProfileType;
  isAuthenticated: boolean;
}

type CustomTrackingUrlMap = {
  [key in AppRoute]+?: (match: Match, trackingConfig: TrackingConfig) => string;
};

export const CustomTrackingUrls: CustomTrackingUrlMap = {
  [AppRoutes['VERGUNNINGEN/DETAIL']]: (match: Match) => {
    return `/vergunning/${match.params?.title}`;
  },
  [AppRoutes['INKOMEN/BBZ']]: (match: Match) => {
    return `/inkomen/bbz`;
  },
  [AppRoutes['INKOMEN/BIJSTANDSUITKERING']]: (match: Match) => {
    return `/inkomen/bijstandsuitkering`;
  },
  [AppRoutes['INKOMEN/TOZO']]: (match: Match) => {
    return `/inkomen/tozo/${match.params?.version}`;
  },
  [AppRoutes['INKOMEN/TONK']]: () => `/inkomen/tonk`,

  [AppRoutes['BURGERZAKEN/IDENTITEITSBEWIJS']]: () =>
    '/burgerzaken/identiteitsbewijs',

  [AppRoutes['ZORG/VOORZIENING']]: () => '/zorg-en-ondersteuning/voorziening',

  [AppRoutes['TOERISTISCHE_VERHUUR/VERGUNNING']]: (match) => {
    return `/toeristische-verhuur/vergunning/${match.params.caseType ?? ''}`;
  },

  [AppRoutes['KLACHTEN/KLACHT']]: () => '/klachten/klacht',

  [AppRoutes['ERFPACHT/DOSSIERDETAIL']]: () => '/erfpacht/dossier',

  [AppRoutes.HOME]: (
    match: Match,
    { profileType, isAuthenticated }: TrackingConfig
  ) => `/${isAuthenticated ? 'dashboard' : 'landing'}`,
};

export function isPrivateRoute(pathname: string) {
  return PrivateRoutes.some(
    (path) =>
      !!matchPath(pathname, {
        path,
        exact: true,
        strict: true,
      })
  );
}
