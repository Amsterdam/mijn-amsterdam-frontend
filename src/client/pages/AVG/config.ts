import { AVGRequestFrontend } from '../../../server/services/avg/types';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

export const displayPropsAanvragen: DisplayProps<
  WithDetailLinkComponent<AVGRequestFrontend>
> = {
  detailLinkComponent: 'Nummer',
  ontvangstDatumFormatted: 'Ontvangen op',
  themas: 'Onderwerp(en)',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => !avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
};

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/privacy/loket/',
    title: 'Loket persoonsgegevens gemeente Amsterdam',
  },
];
