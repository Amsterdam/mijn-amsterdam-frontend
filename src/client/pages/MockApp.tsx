import { RecoilRoot, RecoilRootProps } from 'recoil';
import React from 'react';
import { MemoryRouter, Route } from 'react-router-dom';

interface MockAppProps {
  routePath: string;
  routeEntry: string;
  initializeState?: RecoilRootProps['initializeState'];
  component: React.Component | any;
}

export default function MockApp({
  routePath,
  routeEntry,
  initializeState,
  component,
}: MockAppProps) {
  return (
    <RecoilRoot initializeState={initializeState}>
      <MemoryRouter initialEntries={[routeEntry]}>
        <Route path={routePath} component={component} />
      </MemoryRouter>
    </RecoilRoot>
  );
}
