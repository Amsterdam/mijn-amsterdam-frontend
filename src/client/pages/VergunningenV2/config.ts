export const displayPropsHuidigeVergunningen = {
  identifier: 'Kenmerk',
  title: 'Soort vergunning',
  dateRequest: 'Aangevraagd',
};

export const displayPropsEerdereVergunningen = {
  identifier: 'Kenmerk',
  title: 'Soort vergunning',
  decision: 'Resultaat',
};

export const listPageParamKind = {
  actual: 'huidige-vergunningen-en-ontheffingen',
  historic: 'eerdere-vergunningen-en-ontheffingen',
};

export type ListPageParamKey = keyof typeof listPageParamKind;
export type ListPageParamKind = (typeof listPageParamKind)[ListPageParamKey];

export const listPageTitle = {
  [listPageParamKind.actual]: 'Huidige vergunningen en ontheffingen',
  [listPageParamKind.historic]:
    'Eerdere en niet verleende vergunningen en ontheffingen',
};
