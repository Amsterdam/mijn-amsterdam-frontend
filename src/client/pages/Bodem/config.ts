import { generatePath } from 'react-router-dom';

import { LoodMetingFrontend } from '../../../server/services/bodem/types';
import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';

const displayPropsAanvragen = {
  detailLinkComponent: 'Adres',
  datumAanvraagFormatted: 'Aangevraagd',
  status: 'Status',
};

const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    ...tableConfigBase,
    title: 'Lopende aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => !bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
  },
  [listPageParamKind.completed]: {
    ...tableConfigBase,
    title: 'Afgehandelde aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.completed,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    title: 'Meer informatie over lood in de bodem.',
    to: 'https://www.amsterdam.nl/wonen-leefomgeving/bodem/lood-grond/',
  },
] as const;
