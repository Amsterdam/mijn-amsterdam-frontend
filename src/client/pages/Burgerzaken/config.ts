import { generatePath } from 'react-router-dom';

import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { IdentiteitsbewijsFrontend, LinkProps } from '../../../universal/types';
import { withOmitDisplayPropsForSmallScreens } from '../../components/Table/helpers';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

const displayPropsBase: DisplayProps<
  WithDetailLinkComponent<IdentiteitsbewijsFrontend>
> = {
  detailLinkComponent: 'Type',
  datumAfloopFormatted: 'Geldig tot',
};

const listPageParamKind = {
  identiteitsbewijzen: 'identiteitsbewijzen',
};

const displayProps = withOmitDisplayPropsForSmallScreens(displayPropsBase, [
  'datumAfloopFormatted',
]);

export const tableConfig = {
  [listPageParamKind.identiteitsbewijzen]: {
    title: 'Mijn reisdocumenten',
    displayProps,
    sort: dateSort('datumAfloop', 'desc'),
    listPageRoute: generatePath(AppRoutes['BURGERZAKEN/LIST'], {
      kind: listPageParamKind.identiteitsbewijzen,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/burgerzaken',
    title: 'Overzicht en aanvragen bij burgerzaken',
  },
] as const;

export const backLinkDetailPage = AppRoutes.BURGERZAKEN;
