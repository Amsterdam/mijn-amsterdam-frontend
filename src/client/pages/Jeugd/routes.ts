import { JeugdDetail } from './JeugdDetail';
import { JeugdThemaPagina } from './JeugdThema';
import { FeatureToggle } from '../../../universal/config/feature-toggles';

const baseRoute = '/jeugd';

export const JeugdRoutes = [
  {
    route: baseRoute,
    Component: JeugdThemaPagina,
    isActive: FeatureToggle.zorgnedLeerlingenvervoerActive,
  },
  {
    route: `${baseRoute}/voorziening/:id`,
    Component: JeugdDetail,
    isActive: FeatureToggle.zorgnedLeerlingenvervoerActive,
  },
];
