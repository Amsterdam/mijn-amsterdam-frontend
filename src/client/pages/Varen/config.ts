import { generatePath } from 'react-router-dom';

import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { VarenFrontend } from '../../../server/services/varen/config-and-types';

export const displayPropsAanvragen = {
  detailLinkComponent: '',
  title: 'Omschrijving',
  dateRequestFormatted: 'Aangevraagd',
  status: 'Status',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (vergunning: VarenFrontend) => !vergunning.dateEnd,
    listPageRoute: generatePath(AppRoutes['VAREN/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (vergunning: VarenFrontend) => vergunning.dateEnd,
    listPageRoute: generatePath(AppRoutes['VAREN/LIST'], {
      kind: listPageParamKind.completed,
    }),
    ...tableConfigBase,
  },
};
