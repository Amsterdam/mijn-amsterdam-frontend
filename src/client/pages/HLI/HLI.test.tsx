import { render } from '@testing-library/react';
import { generatePath } from 'react-router-dom';
import { MutableSnapshot } from 'recoil';

import {
  StadspasFrontend,
  StadspasOwner,
} from '../../../server/services/hli/stadspas-types';
import { AppRoutes } from '../../../universal/config/routes';
import { appStateAtom } from '../../hooks/useAppState';
import MockApp from '../MockApp';
import ThemaPaginaHLI from './HLI';
import { AppState } from '../../../universal/types';

const owner: StadspasOwner = {
  initials: '',
  firstname: '',
  lastname: '',
};

const stadspas: StadspasFrontend = {
  urlTransactions: '',
  transactionsKeyEncrypted: '123-xxx-000',
  id: 'stadspas-1',
  passNumber: 123123123,
  passNumberComplete: '0303123123123',
  owner,
  dateEnd: '31-07-2025',
  dateEndFormatted: '31 juli 2025',
  budgets: [],
  balanceFormatted: '',
  balance: 0,
};

const testState = {
  HLI: {
    status: 'OK',
    content: {
      regelingen: [
        {
          id: 'hli-item-1',
          title: 'hli item 1',
          dateDecision: '2020-07-24',
          isActual: true,
          link: {
            to: 'http://example.org/ding',
            title: 'Linkje!! naar hli item 1',
          },
          documents: [],
          steps: [
            {
              id: 'hli-step-2',
              status: 'Besluit',
              datePublished: '2020-08-08',
              description: 'U krijgt HLI Item',
              documents: [],
              isActive: false,
              isChecked: true,
            },
            {
              id: 'hli-step-1',
              status: 'Levering gestart',
              datePublished: '2020-09-24',
              description: 'De levering van uw HLI item is gestart',
              documents: [],
              isActive: true,
              isChecked: true,
            },
          ],
          dateEnd: null,
          dateStart: null,
          displayStatus: 'Levering gestart',
          receiver: 'Naam Persoon 1, Naam Persoon 2',
          decision: 'toegewezen',
        },
      ],
      stadspas: [stadspas],
    },
  },
} as unknown as AppState;

function initializeState(snapshot: MutableSnapshot) {
  snapshot.set(appStateAtom, testState);
}

describe('<HLI />', () => {
  const routeEntry = generatePath(AppRoutes.HLI);
  const routePath = AppRoutes.HLI;

  const Component = () => (
    <MockApp
      routeEntry={routeEntry}
      routePath={routePath}
      component={ThemaPaginaHLI}
      initializeState={initializeState}
    />
  );

  it('Matches the Full Page snapshot', () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
