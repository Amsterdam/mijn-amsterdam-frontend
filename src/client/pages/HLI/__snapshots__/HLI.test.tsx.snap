// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<HLI /> > Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="_Page_47d7c0 _OverviewPage_47d7c0"
  >
    <header
      class="_PageHeading_3b337d _hasIcon_3b337d _hasBackLink_3b337d"
    >
      <span
        aria-hidden="true"
        class="_Icon_3b337d"
      >
        <svg
          aria-label="Stadspas en regelingen bij laag inkomen"
          class="_ThemaIcon_2fc54d"
          fill="black"
          height="32"
          viewBox="0 0 32 32"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M24 0H8V3H24V0ZM28 6H4V9H28V6ZM0 14C0 12.8954 0.895432 12 2 12H30C31.1046 12 32 12.8954 32 14V30C32 31.1046 31.1046 32 30 32H2C0.895432 32 0 31.1046 0 30V14ZM16.375 21.4615V19.8461H12.625C12.7361 19.7424 12.8258 19.6188 12.9189 19.4905C13.0674 19.2858 13.2243 19.0694 13.4904 18.9038C15.2212 17.1538 18.1058 17.1538 19.9808 18.9038L22 17.0192C18.9712 14.3269 14.2115 14.3269 11.3269 17.0192C10.4615 17.8269 9.88461 18.7692 9.59616 19.8461H7V21.4615H9.30769V22.6731H7V24.2885H9.59616C9.88461 25.2308 10.4615 26.1731 11.3269 26.9808C14.3558 29.6731 19.1154 29.6731 22 26.9808L19.9808 25.0962C18.25 26.8462 15.3654 26.8462 13.4904 25.0962C13.2019 24.8269 12.9135 24.5577 12.7692 24.2885H16.375V22.6731H12.0481V21.4615H16.375Z"
            fill-rule="evenodd"
          />
        </svg>
      </span>
      <div
        class="_HeadingInner_3b337d"
      >
        <a
          class="ams-link ams-link--standalone"
          href="/"
        >
          <svg
            aria-hidden="true"
            aria-label="chevron-links-icoon"
            class="_Icon_c3ab45 _Icon__left_c3ab45"
            fill="#000"
            focusable="false"
            height="14px"
            viewBox="0 0 32 32"
            width="14px"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <polygon
              points="22.8571429 32 6.85714286 16 22.8571429 0 25.7568858 2.91002571 12.6771943 16 25.7568858 29.0899743"
            />
          </svg>
          Home
        </a>
        <h2
          class="ams-heading ams-heading--level-1"
        >
          Stadspas en regelingen bij laag inkomen
        </h2>
      </div>
    </header>
    <div
      class="ams-screen ams-screen--wide"
    >
      <div
        class="ams-grid"
      >
        <div
          class="ams-grid__cell ams-grid__cell--span-all"
        >
          <p
            class="ams-paragraph"
          >
            Hieronder ziet u al uw regelingen. Indien u of uw kinderen in bezit zijn van een Stadspas ziet u ook de stadspasgegevens.
          </p>
        </div>
        <div
          class="ams-grid__cell ams-grid__cell--span-all"
        >
          <ul
            class="ams-link-list"
          >
            <li>
              <a
                class="ams-link-list__link"
                href="https://www.amsterdam.nl/werk-inkomen/hulp-bij-laag-inkomen/"
                rel="noreferrer"
              >
                <span
                  class="ams-icon ams-icon--size-5"
                >
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    viewBox="0 0 32 32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="m9.757 32-2.9-2.91L19.937 16 6.857 2.91 9.757 0l16 16z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </span>
                Meer informatie over regelingen
              </a>
            </li>
            <li>
              <a
                class="ams-link-list__link"
                href="https://www.amsterdam.nl/stadspas"
                rel="noreferrer"
              >
                <span
                  class="ams-icon ams-icon--size-5"
                >
                  <svg
                    aria-hidden="true"
                    focusable="false"
                    viewBox="0 0 32 32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="m9.757 32-2.9-2.91L19.937 16 6.857 2.91 9.757 0l16 16z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </span>
                Meer informatie over Stadspas
              </a>
            </li>
          </ul>
        </div>
        <div
          class="ams-grid__cell ams-grid__cell--span-all"
        >
          <ul
            class="ams-unordered-list ams-unordered-list--no-markers"
          >
            <li
              class="ams-unordered-list__item"
            >
              <a
                class="ams-link ams-link--standalone _MaRouterLink__no-underline-fat_aa758a"
              >
                <span
                  class="_Stadspas_owner_2b0bd1"
                />
              </a>
            </li>
          </ul>
          <p
            class="ams-paragraph ams-paragraph--small"
          >
            De stadspas heeft een einddatum van 31 juli 2025
          </p>
        </div>
        <div
          class="ams-grid__cell ams-grid__cell--span-all"
        >
          <h3
            class="ams-heading ams-heading--level-2"
          >
            Huidige regelingen
          </h3>
          <div
            class="ams-table"
          >
            <table
              class="ams-table__table _TableV2_4c6825 _HuidigeRegelingen_2b0bd1"
            >
              <thead
                class="ams-table__header"
              >
                <tr
                  class="ams-table__row"
                >
                  <th
                    class="ams-table__header-cell"
                  >
                    Regeling
                  </th>
                  <th
                    class="ams-table__header-cell"
                  >
                    Naam ontvanger
                  </th>
                </tr>
              </thead>
              <tbody
                class="ams-table__body"
              >
                <tr
                  class="ams-table__row"
                >
                  <td
                    class="ams-table__cell"
                  >
                    <a
                      class="ams-link ams-link--standalone _MaRouterLink__no-underline-fat_aa758a"
                      href="http://example.org/ding"
                    >
                      Hli item 1
                    </a>
                  </td>
                  <td
                    class="ams-table__cell"
                  >
                    Naam Persoon 1, Naam Persoon 2
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div
          class="ams-grid__cell ams-grid__cell--span-all"
        >
          <h3
            class="ams-heading ams-heading--level-2"
          >
            Eerdere en afgewezen regelingen
          </h3>
          <div
            class="ams-table"
          >
            <table
              class="ams-table__table _TableV2_4c6825 _EerdereRegelingen_2b0bd1"
            >
              <tbody
                class="ams-table__body"
              />
            </table>
          </div>
          <p
            class="ams-paragraph"
          >
            U heeft geen eerdere en afgewezen regelingen
          </p>
        </div>
      </div>
    </div>
  </main>
  <p
    class="ams-paragraph _HistoricItemsMention_2b0bd1"
  >
    U ziet hier niet alle gegevens uit het verleden. De gegevens die u hier niet ziet, heeft u eerder per post ontvangen.
  </p>
</DocumentFragment>
`;
